swing/src/net/sf/openrocket/gui/dialogs/preferences/GraphicsPreferencesPanel.java

```diff
@@ -145,7 +145,7 @@ public void stateChanged(ChangeEvent e) {
 		 * we will rely on using the xdg-open command which allows the user to pick
 		 * their preferred application.
 		 */
-		if (SystemInfo.getPlatform() != Platform.UNIX && !SystemInfo.isConfined()) {
+		if (SystemInfo.getPlatform() != Platform.UNIX || !SystemInfo.isConfined()) {
```

Comment:
Just looking at this and reading the comment defining the intention, I feel that this should actually be:

```java
if (SystemInfo.getPlaform() == Platform.UNIX && !SystemInfo.IsConfined()) {
...
```

Reply:
Scratch that - the logic is correct, it just took me a moment.

Essentially, we want to run it we're not on Linux and its not a confined state. Your's does the trick